src/pytest_benchmark/stats.py

```diff
@@ -168,6 +169,7 @@ def __init__(self, fixture, iterations, options):
         self.group = fixture.group
         self.param = fixture.param
         self.params = fixture.params
+        self.cprofile_stats = fixture.cprofile_stats
 
         self.iterations = iterations
         self.stats = Stats()
```

Comment:
Now it becomes really obvious that I picked a confusing name for these two stats attributes. If you have naming ideas let me know (eg: what if it's would be `rawstats` or `data`? or maybe rename `BenchmarkStats` to `BenchmarkData` and so on?)